_includes/releases/v22.2/v22.2.0-alpha.1.md

```diff
@@ -0,0 +1,851 @@
+- Google Cloud KMS will now accept the `gcp-kms` scheme along with the existing `gs` scheme. External Connections will only recognize the `gcp-kms` scheme when being created to represent a KMS resource. [#85957][#85957]
+- The asynchronous garbage collection process has been changed such that quickly after dropping a table, index, or database, or after refreshing a materialized view, the system will issue range deletion tombstones over the dropped data. These tombstones will result in the KV statistics properly counting these bytes as garbage. Before this change, the asynchronous "gc job" would wait out the TTL and then issue a lower-level operation to clear out the data. That meant that while the job was waiting out the TTL, the data would appear in the statistics to still be live. [#85878][#85878]
+- Users can now `CREATE EXTERNAL CONNECTION` to represent an underlying [Google Cloud Storage resource](../{{site.versions["dev"]}}/use-cloud-storage-for-bulk-operations.html). [#85964][#85964]
+- When running `ALTER TABLE ... ADD PRIMARY KEY` or `ALTER TABLE ... ADD CONSTRAINT ... PRIMARY KEY` in a single-statement, implicit transaction, when no primary key had been added to the table, the previous `rowid` column which had been automatically created as the table's `PRIMARY KEY` will now be dropped. [#86195][#86195]
+- Added contention time to `execution_insights`. [#85959][#85959]
+- Secondary regions, like the primary regions, can either be set inside or outside of a [super region](../{{site.versions["dev"]}}/add-super-region.html). In order to move a secondary region `alter_primary_region_super_region_override` must be enabled and the primary region moved. [#84999][#84999]
+- Added support for the `IF EXIST` syntax on `DROP SECONDARY REGION`. Using it will avoid returning an error if a secondary region is not defined on a database. [#84999][#84999]
+- Enabled a new subsystem, `insights`, for gathering slow statement executions in the [`crdb_internal.cluster_execution_insights`](../{{site.versions["dev"]}}/crdb-internal.html) table along with possible reasons for the slowness: full scans or missing indexes, contention, plan changes, retries, etc. This system may be tuned by a handful of new cluster settings and monitored with a handful of new metrics, all in the `sql.insights` namespace. [#86216][#86216]
+- Users can now `CREATE EXTERNAL CONNECTION` to represent an [Azure Storage URI](../{{site.versions["dev"]}}/use-cloud-storage-for-bulk-operations.html). [#86257][#86257]
+- Added support for `SHOW CREATE EXTERNAL CONNECTION` and `SHOW CREATE ALL EXTERNAL CONNECTIONS` that displays the connection name and the unredacted query used to create the external connection. This can only be run by users of the `admin` role today. [#86161][#86161]
```

Comment:
```
Added the `SHOW CREATE EXTERNAL CONNECTION` and `SHOW CREATE ALL EXTERNAL CONNECTIONS` statements, which display the connection name and the unredacted query used to create the external connection. Currently, this can only be run by users of the `admin` role. [#86161][#86161]
```